README.md

```diff
@@ -64,3 +64,21 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+# Running tests
```

Comment:
It would be nice to also mention any dependencies, such as having Firefox and the Geckodriver on your path